wiki/Building-From-Source.md

```diff
@@ -65,14 +80,41 @@ If you want to edit and rebuild the code, you can skip the `prepare-deps` comman
 For the client you need install:
 
 * [Android Studio](https://developer.android.com/studio) or the [sdkmanager](https://developer.android.com/studio/command-line/sdkmanager)
-* Platform Tools 33 (Android 13)
-* Latest NDK (currently v25.1.8937393)
-* Set the environment variable `JAVA_HOME`
-  * For example on Windows: `C:\Program Files\Android\Android Studio\jre`
-* Set the environment variable `ANDROID_SDK_ROOT`
-  * For example on Windows: `%LOCALAPPDATA%\Android\Sdk`
-* Set the environment variable `ANDROID_NDK_HOME`
-  * For example on Windows: `%LOCALAPPDATA%\Android\Sdk\ndk\25.1.8937393`
+* Latest Android SDK Platform-Tools 33 (Android 13)
+* Latest Android NDK (currently v25.1.8937393)
+
+On Linux, the three mentioned developer applications can be installed on Linux from upstream.
+
+* **Arch**
+
+  ```bash
+  sudo pacman -S tandroid-sdk-platform-tools sdkmanager android-ndk
```

Comment:
It's `android-sdk-platform-tools`
I'd just link to the Arch Wiki page though.

Reply:
What about ndk? It isn't mentioned on their wiki page, though

Reply:
How's this, though?

> * **Arch**
>
>    * Packages can vary, read up on the Arch Wiki's [Android](https://wiki.archlinux.org/title/Android#Other_SDK_packages_in_the_AUR) page.

Reply:
> What about the ndk? There's no mention of "ndk" on their wiki page, though
> 
> EDIT: Same going for Ubuntu's AndroidSDK wiki page.

Ah true, add the link but keep the install instructions then